src/data/stream/init.lean

```diff
@@ -0,0 +1,626 @@
+/-
+Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Leonardo de Moura
+-/
+
```

Comment:
Similarly in this file, it would be nice to explain the choice of `init` in the filename.